features/canvas-2d-context.yml

```diff
@@ -0,0 +1,6 @@
+name: Canvas 2D getContextAttributes() method
+description: Returns an object with the attributes used by the context.
+spec: https://html.spec.whatwg.org/multipage/canvas.html#2dcontext:dom-context-2d-canvas-getcontextattributes-2
+group: canvas
+compat_features:
+  - api.CanvasRenderingContext2D.getContextAttributes
```

Comment:
I'm very skeptical this is a distinct feature, rather than a later addition to the canvas API. I'd fold this into the 2D canvas and use a `compute_from` such that this is a later addition.

Reply:
It landed as such a distinctly different time that it doesn't feel like it is part of the initial feature. Yes, it is odd to have a single key like this, but it would be confusing to say it was supported when Canvas 2D first launched, when it wasn't.